dataproc.yaml.tmpl

```diff
@@ -3,6 +3,7 @@ gceClusterConfig:
   zoneUri: ${zone}
   subnetworkUri: ${subnetwork_uri}
   internalIpOnly: true
+  tags: ${yamlencode(dataproc_network_tags)}
```

Comment:
Do you have any examples of how this encodes?

Does it do something like this?
```
tags:
  - tag1
  - tag2
```

Or this:
```
tags: [tag1, tag2]
```

Reply:
The documentation makes me think it would be: